### PR TITLE
adding first pass at buildspec file

### PR DESCRIPTION
**Pull request by RipleyRyan**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -17,12 +17,12 @@ android {
...
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
+//    buildTypes {
+//        release {
+//            minifyEnabled false
+//            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+//        }
+//    }
```
Comment (mtuthill) [inline app/build.gradle:25]: Nitpicky, but do we need to have commented-out stuff still included?

File: app/build.gradle [hunk 2]
```diff
@@ -46,7 +51,7 @@ dependencies {
     testImplementation 'junit:junit:4.+'
     testImplementation 'androidx.test.ext:junit:1.1.0'
     testImplementation 'org.robolectric:robolectric:4.4'
-    testImplementation "org.robolectric:shadows-supportv4:3.8"
+    //testImplementation "org.robolectric:shadows-supportv4:3.8"
```
Comment (mtuthill) [inline app/build.gradle:54]: Nitpicky again, I'd vote to just delete it

File: app/build.gradle (hunk 1 above)
Comment (RipleyRyan) [reply app/build.gradle:25]: No we don't. I'll clean it up a little bit and re-request review, I think there are a few files I created that can be deleted as well. Also need to test and make sure the build fails when it's supposed to.

Comment (mtuthill) [reply app/build.gradle:25]: Sounds good

**Review by mtuthill — APPROVED** [review]

**Review by emilyschroeder20 — APPROVED** [review]

**Pull request merged**